images/linux/toolsets/toolset-1604.json

```diff
@@ -213,5 +213,26 @@
             "3.1",
             "5.0"
         ]
+    },
+    "Clang": {
```

Comment:
Please use lower camel case for naming `clang`, `gcc`

Reply:
Thank you, changed to lower case